### DISCUSSION TITLE
'Introspect' requires minimum platform version 13.0 for the iOS platform, but this target supports 11.0

### DISCUSSION THREAD

**Discussion by MartinOSX** [jevonmao/PermissionsSwiftUI#126] [Q&A]
Hi devs, 
Im getting built failed because of this: 
```
"The package product 'Introspect' requires minimum platform version 13.0 for the iOS platform, but this target supports 11.0".
The iOS deployment target is 15.0.
```
Has anyone else had this problem?

Thanks

<img width="268" alt="Screenshot 2023-02-16 alle 19 13 46" src="https://user-images.githubusercontent.com/74927155/219452372-57451f5b-6caa-4a00-a211-365810f4d47b.png">

**Comment by jevonmao** [comment]
@MartinOSX Did you accidentally do `import CorePermissionsSwiftUI` instead of individual permissions?

**Reply by MartinOSX** [reply]
No, only `PermissionsSwifUIPhoto` and `PermissionsSwiftUICamera`

**Comment by oceanrescue** [comment]
Have the same issue. There are only PermissionsSwifUIPhoto and PermissionsSwiftUICamera had been imported

**Comment by JoeyEamigh** [comment]
Add the package Introspect (https://github.com/siteline/SwiftUI-Introspect) with *exact version* v0.1.4 to your package dependencies and it fixes it.
